packages/web-runtime/src/components/UploadInfo.vue

```diff
@@ -265,6 +272,10 @@ export default defineComponent({
       this.filesInProgressCount += files.filter((f) => !f.isFolder).length
 
       for (const file of files) {
+        if (!this.disableActions && file.source && file.source !== 'DropTarget') {
```

Comment:
```suggestion
        if (!this.disableActions && file?.source !== 'DropTarget') {
```

Reply:
That's actually a different behaviour. `file?.source !== 'DropTarget'` would evaluate to `true` if `file.source` is not given, which I don't want.

Reply:
Yeah, would only be equivalent for `===`